frontend/packages/wizard/src/wizard.tsx

```diff
@@ -161,7 +169,18 @@ const Wizard = ({
         if (params?.toOrigin && origin) {
           navigate(origin);
         } else {
-          dispatch(WizardAction.BACK);
+          dispatch({ type: WizardActionType.BACK });
+        }
+      },
+      onNext: (params: { toOrigin?: boolean; keepSearch?: boolean }) => {
```

Comment:
This and the onBack functions are pretty much identical, so we should probably pull it out into a helper function